R/queue.R

```diff
@@ -67,24 +67,29 @@ Queue <- R6::R6Class("Queue", #nolint
     #' @description
     #' Gets status of packet run
     #'
-    #' @param job_id Id of redis queue job to get status of.
+    #' @param job_ids Id of redis queue job to get status of.
+    #' @param include_logs Whether to include logs in response or not
     #' @return status of redis queue job
-    get_status = function(job_id) {
-      if (!rrq::rrq_task_exists(job_id, controller = self$controller)) {
-        porcelain::porcelain_stop("Job ID does not exist")
+    get_status = function(job_ids, include_logs = TRUE) {
+      valid_job_ids <- job_ids[rrq::rrq_task_exists(job_ids, controller = self$controller)]
+      if (length(valid_job_ids) != length(job_ids)) {
+        cli::cli_warn("Some job ids do not exist in the queue")
```

Comment:
i think you want, or something very similar to this, this will print out exactly what task ids do not exist (note we can keep all our variable names but rich made the point that task id is how rrq talks about this so for user facing messages lets refer to this as task id)
```suggestion
      invalid_job_ids <- job_ids[!rrq::rrq_task_exists(job_ids, controller = self$controller)]
      if (length(invalid_job_ids) != 0) {
        invalid_job_ids_string <- paste0(invalid_job_ids, collapse = ", ")
        cli::cli_warn(
          "Some task ids do not exist in the queue",
          i = paste0("Unknown task ids: ", invalid_job_ids_string)
        )
```